memo/src/lib.rs

```diff
@@ -3,3 +3,6 @@
 //! A simple program that accepts a string of encoded characters and verifies that it parses. Currently handles UTF-8.
 
 pub mod processor;
+
+/// The spl-memo program's on-chain program id
+pub const PROGRAM_ID: &str = "Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo";
```

Comment:
Is this intentionally not a Pubkey type?

Reply:
Nope. Replaced with the `declare_id!` macro. However, CI will fail until this is released: https://github.com/solana-labs/solana/pull/10671
Hoping to get it in 1.2.4